.github/workflows/operate-first-publish.yaml

```diff
@@ -8,42 +8,43 @@ on:
 
 jobs:
   build:
-    name: Update Operate First directory
+    name: Create PR in operate-first/apps repo with updated manifests
     runs-on: ubuntu-latest
     steps:
-    
-      - name: Checkout Operate First repository
-        uses: actions/checkout@v2
-        with:
-          repository: fybrik/operate-first-apps
-          path: operate-first-repo
       - name: Checkout Fybrik repository
         uses: actions/checkout@v2
         with:
           repository: fybrik/fybrik
           path: fybrik-repo
+      - name: Checkout original Operate First repository
+        uses: actions/checkout@v2
+        with:
+          repository: operate-first/apps
+          path: operate-first-repo
       - run: |
-          rm -r fybrik-repo/samples/operate-first/*/
-          rm fybrik-repo/samples/operate-first/*.yaml
+          rm -r fybrik-repo/integration/operate-first/*/
           pushd fybrik-repo/hack/tools && ./install_yq.sh && ./install_helm.sh && popd
-          pushd fybrik-repo/samples/operate-first
+          pushd fybrik-repo/integration/operate-first
           make helm-template
           pip install pipenv
           pipenv install
           pipenv run python filterClusterScoped.py
+          rm ./*.yaml && popd
+          rsync -a --include='*/' --include='*.yaml' --exclude='*' fybrik-repo/integration/operate-first/ operate-first-repo/cluster-scope/base
       - uses: tibdex/github-app-token@v1
         id: generate-token
         with:
-          app_id: ${{ secrets.CHARTS_APP_ID }}
-          private_key: ${{ secrets.CHARTS_APP_PRIVATE_KEY  }}
-          repository: fybrik/fybrik
+          app_id: ${{ secrets.FYBRIK_BOT_APP_ID }}
+          private_key: ${{ secrets.FYBRIK_BOT_PRIVATE_KEY }}
+          repository: fybrik/charts
```

Comment:
I think the repo needs to be `fybrik/operate-first-apps`

Reply:
@revit13 Thanks, just changed it.